design-documents/asynchronous-import/base-extension.md

```diff
@@ -113,6 +161,44 @@ Example:
 }
 ```
 
+### Update Imported Source Format
+
+Its possible also to Update Format 
+
+PUT  `/V1/import/source/:uuid`
```

Comment:
```suggestion
PUT  `/V1/import/source/csv/:uuid`
```

It still makes sense to have dedicated endpoint for validation and strict interface purposes.